scripts/validation/tob_wgs_call_rate_distribution.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+"""Plot call rate distribution for TOB-WGS dataset"""
+
+from bokeh.io.export import get_screenshot_as_png
+from cpg_utils.hail_batch import (dataset_path, init_batch, output_path)
+import hail as hl
+
+MT = dataset_path('mt/v7.mt/')
+
+
+def main():
+    """plot call rate distribution"""
+
+    init_batch()
+    mt = hl.read_matrix_table(MT)
+    samples = mt.s.collect()
+    n_samples = len(samples)
+    mt = hl.experimental.densify(mt)
+    # filter to biallelic loci only
+    mt = mt.filter_rows(hl.len(mt.alleles) == 2)
+    # filter out variants that didn't pass the VQSR filter
+    mt = mt.filter_rows(hl.len(hl.or_else(mt.filters, hl.empty_set(hl.tstr))) == 0)
+    # Calculate call rate and add to mt
+    samples = mt.s.collect()
+    n_samples = len(samples)
+    mt = mt.annotate_rows(call_rate=1-(hl.agg.sum(hl.is_missing(mt.GT))/n_samples))
+    # plot distribution
+    p = hl.plot.histogram(mt.call_rate, range=(0, 1), bins=30, legend='Call Rate', title='Call Rate Distribution')
```

Comment:
Are those enough bins?

Would a `cumulative_histogram` be more informative for this use case? (For finding a threshold.)

Reply:
I tested 50 previously with the same number of variants, and it was a bit combed, so I dropped to 30. I can add in the `cumulative_histogram` as well. That's a good idea